### PR TITLE
Replace apt-key in APT instructions

### PR DESCRIPTION
**Pull request by kingosticks**

### CODE REVIEW

File: docs/installation/debian.rst [hunk 1]
```diff
@@ -42,11 +42,18 @@ Install from apt.mopidy.com
...
+       sudo mkdir -p /usr/local/share/keyrings
+       wget -q -O - https://apt.mopidy.com/mopidy.gpg | gpg --dearmor | \
+         sudo tee /usr/local/share/keyrings/mopidy-archive-keyring.gpg
 
 #. Add the APT repo to your package sources::
 
-       sudo wget -q -O /etc/apt/sources.list.d/mopidy.list https://apt.mopidy.com/buster.list
+       echo "\
+       deb [signed-by=/usr/local/share/keyrings/mopidy-archive-keyring.gpg] \
+       https://apt.mopidy.com/ buster main contrib non-free
```
Comment (djmattyg007) [inline docs/installation/debian.rst:53]: Is this line supposed to have a trailing backslash?

**Comment by djmattyg007** [comment]
Looks good to me, just a minor thing because I wasn't sure.

Half off-topic, I'm so glad that Debian are working to improve this situation 👍

File: docs/installation/debian.rst (hunk 1 above)
Comment (kingosticks) [reply docs/installation/debian.rst:53]: No, because I want a new-line before the `deb-src` definition that follows.

**Comment by kingosticks** [comment]
> Half off-topic, I'm so glad that Debian are working to improve this situation 

I wish there had been a way to incorporate this into apt-key somehow and reduce the friction.

**Review by djmattyg007 — APPROVED** [review]

File: docs/installation/debian.rst [hunk 2]
```diff
@@ -42,11 +42,18 @@ Install from apt.mopidy.com
...
+       deb-src [signed-by=/usr/local/share/keyrings/mopidy-archive-keyring.gpg] \
+       https://apt.mopidy.com/ buster main contrib non-free" | \
+         sudo tee /etc/apt/sources.list.d/mopidy.list
```
Comment (jodal) [inline docs/installation/debian.rst:56]: Cant't we just update https://apt.mopidy.com/buster.list instead of moving this to the docs?

File: mopidy/config/types.py [hunk 3]
```diff
@@ -279,7 +279,7 @@ class Port(Integer):
 
     def __init__(self, choices=None, optional=False):
         super().__init__(
-            minimum=0, maximum=2 ** 16 - 1, choices=choices, optional=optional
+            minimum=0, maximum=2**16 - 1, choices=choices, optional=optional
```
Comment (jodal) [inline mopidy/config/types.py:282]: This is very unrelated and should be another PR.

Comment (kingosticks) [reply mopidy/config/types.py:282]: What about a pr that removes flake8 and black running on the master branch Ci? I'm getting tired of these.

File: docs/installation/debian.rst (hunk 2 above)
Comment (kingosticks) [reply docs/installation/debian.rst:56]: I figured since the first command allows them to (inadvertently?) use some other directory, we'd be better off making them both explicit. And I have no means to update buster.list

Comment (kingosticks) [reply docs/installation/debian.rst:56]: But I am all for keeping this in Buster.list, if we think that's acceptable.

File: mopidy/config/types.py (hunk 3 above)
Comment (djmattyg007) [reply mopidy/config/types.py:282]: > What about a pr that removes flake8 and black running on the master branch Ci?

It would be better to pin the versions we're using and set up Dependabot or Renovatebot to keep them up to date for us. IMO using poetry would make this easier.

Comment (kingosticks) [reply mopidy/config/types.py:282]: Or that. To be clearer, I am only talking about doing this on master branch.

File: docs/installation/debian.rst (hunk 2 above)
Comment (jodal) [reply docs/installation/debian.rst:56]: `buster.list` is managed in the `apt` repo. It is deployed on commit.

Comment (kingosticks) [reply docs/installation/debian.rst:56]: Oh cool ok. So we'll go for that then?

**Comment by kingosticks** [comment]
OK, got around to moving the other change to https://github.com/mopidy/apt/pull/12

**Review by djmattyg007 — APPROVED** [review]
